crates/db_views/site/src/api.rs

```diff
@@ -732,7 +743,12 @@ pub struct UserSettingsBackup {
   #[serde(default)]
   pub blocked_users: Vec<Url>,
   #[serde(default)]
+  /// Only kept for v0.19 compatibility
   pub blocked_instances: Vec<String>,
+  #[serde(default)]
+  pub blocked_instances_communities: Vec<String>,
```

Comment:
It doesnt make any sense to keep `blocked_instances` without using it. That means old backups imported to 1.0 will not work properly as blocked communities will be missing. You either need to get rid of `blocked_instances_communities` and instead use `blocked_instances` like before, or do like this:
```
  #[serde(default, rename = "blocked_instances")]
  pub blocked_instances_communities: Vec<String>,
```